cves/kernel/CVE-2016-9754.yml

```diff
@@ -389,38 +389,38 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
```

Comment:
Consider adding a note for why it's false. Other documents I've reviewed included this, so it might be good to have it.